JavaScript/JavaScriptSDK/Context/Operation.ts

```diff
@@ -12,7 +12,10 @@ module Microsoft.ApplicationInsights.Context {
         public syntheticSource: string;
 
         constructor() {
-            this.id = Util.newGuid();
+            this.id = Util.newGuid();            
+            if (window && window.location && window.location.pathname) {
+                this.name = window.location.pathname;
```

Comment:
I understand it. However 
1) by default page name equals to document.title so if I set operation name being page name it won't be usable;
 and 
2) there's no explicit connection between this and operation name. So I would not expect change in page name impacting operation name.
I don't see the value in hiding URL, but if this is the case then users will need to override it manually.